ifplugd: Don't leak FDs.

These don't need to end up in any spawned children.  Set cloexec so
they don't leak.

--- a/networking/ifplugd.c
+++ b/networking/ifplugd.c
@@ -603,9 +603,11 @@ int ifplugd_main(int argc UNUSED_PARAM,
 		bb_daemonize_or_rexec(DAEMON_CHDIR_ROOT, argv);
 
 	xmove_fd(xsocket(AF_INET, SOCK_DGRAM, 0), ioctl_fd);
+	close_on_exec_on(ioctl_fd);
 	if (opts & FLAG_MONITOR) {
 		int fd = create_and_bind_to_netlink(NETLINK_ROUTE, RTMGRP_LINK, 0);
 		xmove_fd(fd, netlink_fd);
+		close_on_exec_on(netlink_fd);
 	}
 
 	write_pidfile(pidfile_name);
